test/test_set_poly_reptype.py

```diff
@@ -0,0 +1,26 @@
+import cdd
+import numpy as np
+
+def test_set_float_polyhedron_rep_type():
+    A = np.array([[1, 0, 1], [1, 1, 0], [1, 1, 1], [1, 0, 0]])
```

Comment:
Why a numpy array and not a simple list of lists? That would allow dropping the numpy dependency. Also instead of introducing a local variable A, you can feed it directly into cdd.Matrix as it isn't used elsewhere.